app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -0,0 +1,170 @@
+    isLoading (newLoading, _priorLoading) {
+      if (newLoading || !this.clanIdOrSlug) {
+        return
+      }
+      // This allows us to always use id to work with currently selected clan.
+      this.findIdOfParam()
+    }
+  },
+
+  created () {
+    this.clanIdOrSlug = this.$route.params.idOrSlug || null
+    this.findIdOfParam()
+    // Would be odd to arrive here with ?registering=true and be logged out...
+    this.leagueSignupModalOpen = this.canRegister && !!this.$route.query.registering
+  },
+
+  methods: {
+    ...mapActions({
+      loadClanRequiredData: 'seasonalLeague/loadClanRequiredData',
+      loadGlobalRequiredData: 'seasonalLeague/loadGlobalRequiredData',
+    }),
+
+    changeClanSelected (e) {
+      if (e.target.value === 'global') {
+        this.clanIdOrSlug = ''
+        this.clanIdSelected = ''
+      } else {
+        this.clanIdOrSlug = e.target.value
+      }
+      this.findIdOfParam()
+
+      application.router.navigate(`league/${this.clanIdSelected}` , { trigger: true })
+    },
+
+    findIdOfParam () {
+      if (this.clanIdOrSlug) {
+        this.clanIdSelected = (this.clanByIdOrSlug(this.clanIdOrSlug) || {})._id
+        this.loadClanRequiredData({ leagueId: this.clanIdSelected })
+      } else {
+        this.loadGlobalRequiredData()
+      }
+    },
+
+    signupAndRegister () {
+      window.nextURL = '/parents?registering=true'
+      application.router.navigate('?registering=true', { trigger: true })
+    },
+
+    async submitRegistration (registration) {
+      // TODO: isRegistered is not reactive because we're not using Vuex :( Improve this
+      this.doneRegistering = true
+
+      // TODO: Validate here too?
+      me.set('firstName', registration.firstName)
+      me.set('lastName', registration.lastName)
+      me.set('name', registration.name)
+      me.set('email', registration.email)
+      me.set('emails', registration.emails)
+      me.set('birthday', registration.birthday)
+      me.set('unsubscribedFromMarketingEmails', registration.unsubscribedFromMarketingEmails)
+      await me.save()
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      globalRankings: 'seasonalLeague/globalRankings',
+      clanRankings: 'seasonalLeague/clanRankings',
+      myClans: 'clans/myClans',
+      clanByIdOrSlug: 'clans/clanByIdOrSlug',
+      isLoading: 'clans/isLoading'
+    }),
+
+    currentSelectedClan () {
+      return this.clanByIdOrSlug(this.clanIdSelected) || null
+    },
+
+    selectedClanRankings () {
+      return this.clanRankings(this.clanIdSelected)
+    },
+
+    isRegistered () {
+      const emails = me.get('emails') || {}
+      const unsubscribed = me.get('unsubscribedFromMarketingEmails')
+      return (emails.generalNews || {}).enabled && !unsubscribed
+    },
+
+    canRegister: function () {
+      return !me.isAnonymous() && !this.isRegistered
+    },
+
+    // NOTE: `me` and the specific `window.me` are both unavailable in this template for some reason? Hacky...
```

Comment:
What do you mean unavailable in the template? @codeluggage 

Are you referring to between `{{ <code> }}` in the html section?